docker/1.14/Dockerfile.gpu

```diff
@@ -2,7 +2,6 @@ FROM nvidia/cuda:10.0-base-ubuntu16.04
 
 LABEL maintainer="Amazon AI"
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
-LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
1. not clear what this change has to do with ping
2. does this mean we are removing the mme feature from this version? that's a breaking change.